### PR TITLE
Updated namedCustomPlaybooks docs

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: docs/playbook-reference/defining-playbooks/playbook-basics.rst [hunk 1]
```diff
@@ -88,6 +88,43 @@ In order to override the default configuration of the same playbook, both disabl
...
+
+
+.. code-block:: yaml
+
+    namedCustomPlaybooks:
+    team-a-app-a:
+      - triggers:
+          - on_prometheus_alert:
+              namespace_prefix: "app-a"
+      actions:
```
Comment (aantn) [inline docs/playbook-reference/defining-playbooks/playbook-basics.rst:108]: Is the indentation on this correct? Shouldn't this be aligned with triggers?

File: helm/robusta/values.yaml [hunk 2]
```diff
@@ -147,27 +147,7 @@ priorityBuiltinPlaybooks:
...
-#         aggregation_key: "This is app-a - Requires your attention"
-#         severity: HIGH
-#         title: "Check app-a out"
-#         description: "@monitoring.monitoring this is for you"
-#   team-b-app-b:
-#   # Default Settings for all actions
-#   - triggers:
-#     - on_prometheus_alert:
-#         namespace_prefix: "app-b"
+# use to load custom playbooks from other files. Find out more details https://docs.robusta.dev/master/playbook-reference/defining-playbooks/playbook-basics.html#organizing-playbooks
```
Comment (aantn) [inline helm/robusta/values.yaml:150]: I suggest:

> load custom playbooks by name, which allows overriding them when using multiple Helm values files - learn more https://docs.robusta.dev/master/playbook-reference/defining-playbooks/playbook-basics.html#organizing-playbooks

File: docs/playbook-reference/defining-playbooks/playbook-basics.rst [hunk 3]
```diff
@@ -88,6 +88,43 @@ In order to override the default configuration of the same playbook, both disabl
       - image_pull_backoff_reporter: {}
 
 
+Organizing Playbooks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (aantn) [inline docs/playbook-reference/defining-playbooks/playbook-basics.rst:92]: For the next two lines I suggest a slightly different explanation:

Using ``namedCustomPlaybooks``, you can define playbooks by name. This is useful when you want to define a base set of playbooks for all clusters/teams and then use additional Helm values files to override some of the base playbooks or add new ones.

**Review by aantn — APPROVED** [review]
Left some comments. Please merge after looking.

File: docs/playbook-reference/defining-playbooks/playbook-basics.rst (hunk 1 above)
Comment (arikalon1) [reply docs/playbook-reference/defining-playbooks/playbook-basics.rst:108]: fixed

**Pull request merged**